### PR TITLE
db should work

### PR DESCRIPTION
**Pull request by Eliza-shoe**

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -1,14 +1,93 @@
...
+
+
+db.create_all()
+
 
-app = Flask(__name__)
 @app.route("/")
 def home():
     return render_template("home.html")
-#Test
```
Comment (Eliza-shoe) [inline server/app.py:8]: Whoops shouldn't have changed this. I'll update it in my local files

**Review by Eliza-shoe — COMMENTED** [review]
Database should work correctly.
Features: 
User Class
Chat room class
Restaurant Class
Many to many table connecting user and chat rooms so one user can be in multiple chat rooms and a chat room can have multiple users

File: server/app.py (hunk 1 above)
Comment (nh124) [reply server/app.py:8]: This looks ok.

**Review by nh124 — APPROVED** [review]
This looks ok.

**Pull request merged**